--- github2-0.6.2/setup.py~	2012-05-23 07:11:05.000000000 +0200
+++ github2-0.6.2/setup.py	2014-10-21 20:44:07.711038630 +0200
@@ -27,14 +27,7 @@
 elif sys.version_info[:2] < (2, 6):
     install_requires.append('simplejson >= 2.0.9')
 
-# dateutil supports python 2.x in dateutil-1, python 3.x in dateutil-2.0 and
-# python 2.6+ in dateutil-2.1.  Exciting…
-if sys.version_info[:2] <= (2, 5):
-    install_requires.append('python-dateutil < 2.0')
-elif sys.version_info < (3, ):
-    install_requires.append('python-dateutil < 2.0, >= 2.1')
-else:
-    install_requires.append('python-dateutil > 2.0')
+install_requires.append('python-dateutil >= 2.2')
 
 long_description = (codecs.open('README.rst', "r", "utf-8").read()
     + "\n" + codecs.open('NEWS.rst', "r", "utf-8").read())
--- github2-0.6.2/github2.egg-info/requires.txt~	2014-10-21 20:40:49.000000000 +0200
+++ github2-0.6.2/github2.egg-info/requires.txt	2014-10-21 20:44:39.267704182 +0200
@@ -1,2 +1,2 @@
 httplib2 >= 0.7.0
-python-dateutil < 2.0, >= 2.1
\ No newline at end of file
+python-dateutil >= 2.2
